database/migrations/2024_05_15_042959_add_admin_role_to_users.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->boolean('is_admin')->default(false);
```

Comment:
Idea: if we made this an `int`, we could enumerate "levels" of admins, i.e. `0` would be a plain user, `1` would be a "super admin" and when needed, we could start adding more values to model some sort of permissions.

Reply:
I have no objections against using a boolean for now though, so feel free to disregard this.